flight/boundary/plotter.py

```diff
@@ -0,0 +1,58 @@
+"""
+Provides plotting functionality for visualizing coordinate data
+"""
+
+from typing import List, Dict, Tuple
```

Comment:
In Python 3.10, you don't need to import list, dict, or tuple from typing. Instead, just type with the lowercase versions

Reply:
Removed the import and typed with lowercase.